src/som/primitives/SymbolPrimitives.java

```diff
@@ -43,22 +47,31 @@ public void installPrimitives() {
     installInstancePrimitive(new SPrimitive("asString", universe) {
 
       @Override
-      public void invoke(final Frame frame, final Interpreter interpreter) {
-        SSymbol self = (SSymbol) frame.pop();
-        frame.push(universe.newString(self.getEmbeddedString()));
+      public void invoke(final VirtualFrame truffleFrame,
+          final Interpreter interpreter) throws FrameSlotTypeException {
+        SSymbol self = (SSymbol) StackUtils.pop(truffleFrame);
+
+        SString value = universe.newString(self.getEmbeddedString());
+
+        StackUtils.push(truffleFrame, value);
       }
     });
 
     installInstancePrimitive(new SPrimitive("=", universe) {
 
       @Override
-      public void invoke(final Frame frame, final Interpreter interpreter) {
-        SAbstractObject op1 = frame.pop();
-        SSymbol op2 = (SSymbol) frame.pop(); // self
+      public void invoke(final VirtualFrame truffleFrame,
+          final Interpreter interpreter) throws FrameSlotTypeException {
+
+        SAbstractObject op1 = StackUtils.pop(truffleFrame);
+        SSymbol op2 = (SSymbol) StackUtils.pop(truffleFrame); // self
+
         if (op1 == op2) {
-          frame.push(universe.trueObject);
+
+          StackUtils.push(truffleFrame, universe.trueObject);
         } else {
-          frame.push(universe.falseObject);
+
```

Comment:
There's a lot of change noise everywhere.
A lot of empty lines, and renamings which do not necessarily clarify the code (`truffleFrame`)